smoke-tests/src/test/groovy/io/opentelemetry/smoketest/WildflySmokeTest.groovy

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.opentelemetry.smoketest
+
+
+import okhttp3.Request
+
+class WildflySmokeTest extends SmokeTest {
+
+  protected String getTargetImage() {
+    "jboss/wildfly:latest"
+  }
+
+  //We don't have support for Wildfly Undertow server yet.
+  //So this test just verifies that Wildfly has come up.
```

Comment:
i think this should work now with real span verification, the reason it didn't before is because we were verifying from logging exporter and wildfly messed with stdout/stderr, i can try it out after the merge

Reply:
I haven't seen any spans generated during this test